src/OperationalInsights/OperationalInsights/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fixed bug [#12135]
```

Comment:
Please provide more detail. It's too general. For instance,
Fixed the issue that object cannot be created when .... [#12154 ]